vra/resource_project.go

```diff
@@ -20,11 +20,13 @@ func resourceProject() *schema.Resource {
 			"administrators": {
 				Type:        schema.TypeSet,
 				Optional:    true,
+				Deprecated:  "To specify the type of principal, please refer administrator.",
 				Description: "List of administrator users associated with the project. Only administrators can manage project's configuration.",
 				Elem: &schema.Schema{
 					Type: schema.TypeString,
 				},
 			},
+			"administrator": userSchema(),
```

Comment:
We should have a description for this property

Reply:
Rename to `administrator_roles`